Bottom: 246b218c56e6adfca290ef2abf330da1439f0d14
Top:    790547ea371563fb00399e3f4b5690993e27081c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 23:27:15 +0000

Fix borast not to build for non-GL enabled builds


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 2b50699..fe0f4e5 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -38,22 +38,6 @@ PCB_SRCS = \
 	box.h \
 	buffer.c \
 	buffer.h \
-	borast/borast-minimal.h \
-	borast/borastint-minimal.h \
-	borast/borast-bentley-ottmann.c \
-	borast/borast-combsort-private.h \
-	borast/borast-compiler-private.h \
-	borast/borast-fixed-private.h \
-	borast/borast-fixed-type-private.h \
-	borast/borast-freelist.c \
-	borast/borast-freelist-private.h \
-	borast/borast-malloc-private.h \
-	borast/borast-traps.c \
-	borast/borast-traps-private.h \
-	borast/borast-types-private.h \
-	borast/borast-wideint.c \
-	borast/borast-wideint-private.h \
-	borast/borast-wideint-type-private.h \
 	change.c \
 	change.h \
 	clip.c \
@@ -195,8 +179,23 @@ GL_SRCS= \
 	hid/common/hidgl.c \
 	hid/common/hidgl.h \
 	hid/common/trackball.c \
-	hid/common/trackball.h
-
+	hid/common/trackball.h \
+	borast/borast-minimal.h \
+	borast/borastint-minimal.h \
+	borast/borast-bentley-ottmann.c \
+	borast/borast-combsort-private.h \
+	borast/borast-compiler-private.h \
+	borast/borast-fixed-private.h \
+	borast/borast-fixed-type-private.h \
+	borast/borast-freelist.c \
+	borast/borast-freelist-private.h \
+	borast/borast-malloc-private.h \
+	borast/borast-traps.c \
+	borast/borast-traps-private.h \
+	borast/borast-types-private.h \
+	borast/borast-wideint.c \
+	borast/borast-wideint-private.h \
+	borast/borast-wideint-type-private.h
 
 BUILT_SOURCES = \
 	core_lists.h \
